moped-database/views/project_list_view.sql

```diff
@@ -158,7 +155,15 @@ AS WITH project_person_list_lookup AS (
      LEFT JOIN moped_users added_by_user ON mp.added_by = added_by_user.user_id
      LEFT JOIN current_phase_view current_phase on mp.project_id = current_phase.project_id
      LEFT JOIN moped_public_process_statuses mpps ON mpps.id = mp.public_process_status_id
-     left join child_project_lookup cpl on cpl.parent_id = mp.project_id
+     LEFT JOIN child_project_lookup cpl on cpl.parent_id = mp.project_id
+     LEFT JOIN LATERAL
```

Comment:
LATERAL joins! this is a big TIL for me. i am going to revisit the component view in [this PR](https://github.com/cityofaustin/atd-moped/pull/1148) and see if it can be simplified

Reply:
big TIL for me too, (although every time i write SQL it feels like a TIL)
it was interesting to learn how to select more on than one column from a subquery

Reply:
add me to the TIL train! 🚂